git.md

```diff
@@ -195,6 +195,26 @@ Fix nasty little bug ISSUE-51 #resolve #time 2h30m
 * Keep the history *clean* and *simple*. Before pushing any branch to the
   public always ensure that it conforms to the style-guide. If it doesn't:
   squash, rebase, reword commits as necessary.
+  
+* When merging multiple small commits into a single change, make sure to
+  keep high locality of the changes. In another words, if you're changing,
+  let's say, some elements of the UI, group changes
+  that occur in small vicinity of each other.
+
+* When merging multiple changes into one, make sure to _do not exceed
```

Comment:
Myślałem że mamy po jednej zmianie zazwyczaj na commit?

Reply:
Mamy, ale batch commity zdarzaja sie i beda sie zdarzac. Realistycznie, nie wymagalabym od nikogo osobno commitowania zmiany paddingu. To by wrecz zasmiecalo repo. Jezeli ktos ma pare drobnych poprawek ktore sa ze soba logicznie powiazane, to chcialbym batch commit.

Reply:
To dodałbym "do that only with changes that do not exceed a few lines"